TWLight/resources/templates/resources/csv_form.html

```diff
@@ -1,26 +1,18 @@
 {% extends 'admin/base.html' %}
-{% load i18n %}
 
 {% block content %}
-  {% comment %} Translators: This text is shown to staff who are uploading access codes, instructing them on the format the file should take. {% endcomment %}
-  {% blocktrans trimmed %}
-    To upload access codes, create a .csv file containing two columns: 
-    The first containing the access codes, and the second the ID of the 
-    partner the code should be linked to.
-  {% endblocktrans %}
   <form method="POST" enctype="multipart/form-data" class="form-horizontal">
-  {% csrf_token %}
-  <div class="form-group">
-    {% comment %} Translators: This labels a form where staff upload a file containing access codes. {% endcomment %}
-    <label for="name" class="col-md-3 col-sm-3 col-xs-12 control-label">File: </label>
-    <div class="col-md-8">
-      <input type="file" name="access_code_csv" id="access_code_csv" required="True" class="form-control">
+    {% csrf_token %}
+    <div class="form-group">
+      <label for="name" class="col-md-3 col-sm-3 col-xs-12 control-label">File: </label>
+      <div class="col-md-8">
+        <input type="file" name="access_code_csv" id="access_code_csv" required class="form-control">
+      </div>
     </div>
-  </div>
-  <div class="form-group">
-    <div class="col-md-3 col-sm-3 col-xs-12 col-md-offset-3" style="margin-bottom:10px;">
-      {% comment %} Translators: This labels a button for uploading files containing lists of access codes. {% endcomment %}
-      <button class="btn btn-primary">{% trans "Upload" %}</button>
+    <div class="form-group">
+      <div class="col-md-3 col-sm-3 col-xs-12 col-md-offset-3" style="margin-bottom:10px;">
+        <button class="btn btn-primary">Upload</button>
+      </div>
     </div>
-  </div>
+  </form>
 {% endblock %}
```

Comment:
files should end with newline

Reply:
interestingly, I can see now that you didn't introduce the newline. I swear on the diff view it previously looked as if that was new.